src/Components/SmartComponents/CVEDetailsPage/CVEDetailTableTitle.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+import { Skeleton, Text, TextContent, TextVariants } from '@patternfly/react-core';
+import { useGetSingleDevice } from '../../../Helpers/APIHelper';
+import messages from '../../../Messages';
+
+const CVEDetailTableTitle = () => {
+    const intl = useIntl();
```

Comment:
I do remember @leSamo [initiative](https://redhat-internal.slack.com/archives/C02T90PLJ5V/p1693474473912109) to deprecate but do not remember the final agreement. I think we need to start formalizing our agreements in the handbook. Because it is easy to forget about them later, especially if they are about things that are not in a daily routine.

Reply:
Yeah, this has been mostly between a few people, that is true. But feel free to use plain strings. There is no need to write intl messages anymore (as comes from the recent discussions).

Reply:
Yup, the last commit includes intl removal